core/evancore.vue.libs/src/components/profile/profile-picture/profile-picture.scss

```diff
@@ -20,22 +20,20 @@
 @import '~@evan.network/ui/src/style/utils';
 
 @mixin masks($size) {
```

Comment:
I'm asking myself currently, why the `--size` was used before, makes no sense at all, by using css vars, because the mixin is applied to all the size classes.
In case of a component, the classes will not be used outside. To reduce the amount of generated scss, we could apply the design in general to the components base design.

```scss
& {
  width: var(--size);
  height: var(--size);
....
}
```

The `--size` css variable could be set by the template and would reduce the amount of generated css by 66%.

```vue
:style="`--size: getSizeValue()`"
```

Do we want to do it? ;)

Reply:
I don't quite understand the details, but I would prefer a property like 
`diameter="24"` instead of directly modifying the style.